libnetwork/network.go

```diff
@@ -510,15 +510,6 @@ func (n *Network) CopyTo(o datastore.KVObject) error {
 	return nil
 }
 
-func (n *Network) DataScope() string {
-	s := n.Scope()
-	// All swarm scope networks have local datascope
-	if s == scope.Swarm {
-		s = scope.Local
-	}
-	return s
-}
```

Comment:
Have been staring at some of these various times yes; "scope", "datascope" and "let's juggle between local, swarm, and global" was really confusing!

Good riddance!